packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -350,7 +359,7 @@ export class LionInputDatepicker extends OverlayMixin(LionInputDate) {
     if (this._cachedOverlayContentNode) {
       return this._cachedOverlayContentNode;
     }
-    this._cachedOverlayContentNode = this.shadowRoot.querySelector('lion-calendar-overlay-frame');
+    this._cachedOverlayContentNode = this.shadowRoot.querySelector('.calendar__overlay-frame');
```

Comment:
I would argue to not use class selectors (styles) for functional purposes (element references): https://css-tricks.com/stop-using-css-selectors-non-css/

Especially in our Lion layer, they can be swapped out by Sub classers.
For a practical example: see 'An extension implementation' here: https://github.com/ing-bank/lion/issues/591

Reply:
I didn't see an alternative, I am aware classes are as a best practice for styling only, but as with all best practices, you only follow them if you can. If you can't you need to go with the lesser of multiple evils, which is either id or class, or do you propose something else?

Soon, scoped-elements will release a helper to do query selection on scoped elements by allowing to get the tagname from the scoped registry so we may use that, but I had to patch this somehow in the meantime

Reply:
Let's go for the tag name from scoped-elements indeed. In other contexts, I would use a data attr (or alternatively a class with 'js' prefix) like the article suggests